### ISSUE TITLE
Capture more examples

### ISSUE THREAD

**Issue by nomandera** [nitnelave/lldap#124]
This project is really interesting and IMO will fill a void for self hosters. However from the "outside looking in" the majority of exposure to how well it works is people raising Issues for certain applications.

I strongly suspect this is painting a false picture of how lldap can be used.

This could be remedied by capturing success stories somewhere and ideally expanding the setup guides for applications. It is likely a few dozens applications would cover the majority of the popular self hosters estate and do a lot of the project.

Keep up the good work.

**Comment by nitnelave** [comment]
Yep, that's a work in progress :) I ask people who integrate with LLDAP to post a sample config, there's a folder for that.
I have a couple of things planned, but then I'll have a look at whatever services I run myself, make sure they work well with LLDAP and add their configs to the list.

However, if you have any configuration to contribute, you're welcome to do so!

**Comment by nitnelave** [comment]
Closing this as we have a bunch more configs now, and keep adding more.
